sdk/pom.xml

```diff
@@ -134,16 +134,6 @@
       <version>1.7</version>
       <scope>test</scope>
     </dependency>
-    <dependency>
```

Comment:
@artur-ciocanu are these not used?

Reply:
These are duplicates that Maven was complaining about.